test/test_config.go

```diff
@@ -37,6 +39,13 @@ func MustManager(ctx context.Context, ns string) manager.Manager {
 		Cache: cache.Options{
 			DefaultNamespaces: map[string]cache.Config{ns: cache.Config{}},
 		},
+		Controller: config.Controller{
```

Comment:
I could ❤️ this comment any more. Super useful to have the c-r issue, and the rationale for why we skip it